wasm/solana-client/Cargo.toml

```diff
@@ -22,8 +22,8 @@ reqwest = { version = "0.11.9", features = ["json"] }
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
-solana-extra-wasm = { path = "../utils/solana-extra", version = "=1.14.16" }
-solana-sdk = "=1.14.16"
+solana-extra-wasm = { path = "../utils/solana-extra", version = ">=1.14, <1.17" }
```

Comment:
This is a local dependency and I think both `solana-client-wasm` and `solana-extra-wasm` should continue follow the same versioning as the official Solana crates.